code/modules/surgery/organs/lungs.dm

```diff
@@ -39,6 +39,10 @@
 	var/healium_para_min = 3
 	///Minimum amount of healium to knock you down for good
 	var/healium_sleep_min = 6
+	///Minimum amount of hexane needed to start having effect
+	var/hexane_min = 2
```

Comment:
I'm gonna assume this is the result of merge conflict memes

Reply:
my bad on this one, woops!